bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/WriteFlag.java

```diff
@@ -20,14 +20,22 @@
  */
 package org.apache.bookkeeper.client.api;
 
-import com.google.common.base.Preconditions;
+import static com.google.common.base.Preconditions.checkArgument;
+
 import java.util.EnumSet;
+import lombok.Getter;
 
 /**
  * Flags to specify the behaviour of writes
  */
+@Getter
 public enum WriteFlag {
-    DEFERRED_FORCE(1);
+
+    /**
+     * Writes will be ackknowledged by the server even if the entry
```

Comment:
typo - acknowledged

I would suggest changing the comment to "acknowledged after writing to filesystem but not yet been persisted to disks" or something like that.

Reply:
done